test/CallbackStreamTest.php

```diff
@@ -89,13 +89,13 @@ public function testGetSize()
         $this->assertNull($ret);
     }
 
-    /**
-     * @expectedException RuntimeException
```

Comment:
Ideally we should try to avoid unrelated changes; as far as I know, there's nothing broken about `@expectedException` is there?